app/models/manageiq/providers/ibm_cloud/inventory/collector/vpc.rb

```diff
@@ -115,7 +115,15 @@ def volume_profiles
   end
 
   def tags_by_crn(crn)
-    vpc.cloudtools.tagging.collection(:list_tags, :attached_to => crn, :providers => ["ghost"]).to_a
+    retried = false
+    begin
+      vpc.cloudtools.tagging.collection(:list_tags, :attached_to => crn, :providers => ["ghost"]).to_a
+    rescue IBMCloudSdkCore::ApiException
```

Comment:
What was the exception when we hit a timeout?  I think we should check that the exception was the api request limit error versus something like a 403 forbidden

Reply:
```
[----] E, [2022-12-02T15:52:13.534524#7:977c]   ERR -- evm: MIQ(ManageIQ::Providers::IbmCloud::VPC::CloudManager::Refresher#refresh) EMS: [IBM-CVPC-auto-created], id: [6] Refresh failed
[----] E, [2022-12-02T15:52:13.534822#7:977c]   ERR -- evm: [IBMCloudSdkCore::ApiException]: Error: , Code: , Information: You must wait 35068 ms before you can make ghost-tags get api requests with attached_to query parameter, X-global-transaction-id: gst-bfecd060-c20c-44af-96a7-46bd53b8425f  Method:[block (2 levels) in <class:LogProxy>]
```

Reply:
Okay so we could do `err.message.match?(/You must wait \d+ ms before you can make ghost-tags get api requests/)`